src/__tests__/proposalVersions.int.test.ts

```diff
@@ -140,22 +139,22 @@ describe('/proposalVersions', () => {
 				})
 				.expect(201);
 			const after = await loadTableMetrics('proposal_field_values');
-			logger.debug('after: %o', after);
+
 			expect(before.count).toEqual(0);
 			expect(result.body).toMatchObject({
 				id: 1,
 				proposalId: 1,
 				fieldValues: [
 					{
-						id: 1,
+						id: expect.any(Number) as number,
```

Comment:
this is a cool function!

Reply:
It's cool until you see the output when it matcheth not. But I agree it's better to "expect this to vary" when it really is expected to vary.